.github/workflows/build-jar.yml

```diff
@@ -1,20 +1,25 @@
-name: Build jar with dependencies
+name: Build jar with Dependencies
```

Comment:
Why Dependencies with capital D?

Reply:
I don't like english capitalization rules. That's why. Most of these are no real sentences. Sometimes nouns are capitalized in english, if it is a headline. Are these headlines or sentences? Or something different? I don't know.

Reply:
Doesn't matter. I need it to be merged to continue with the ppa issue.